referencesrv/tests/unittests/test_referencesrv_resolver.py

```diff
@@ -82,11 +82,11 @@ def test_normalize_author_list(self):
         """
         Ensure that authorString is returned in the form AuthorLast1; AuthorLast2
         """
-        self.assertEqual(normalize_author_list("Foo, N.A., Bar, C. K. "), 'Foo, N.A.; Bar, C. K.')
-        self.assertEqual(normalize_author_list("N.A. Foo and C. K. Bar"), 'Foo, N.A.; Bar, C. K.')
-        self.assertEqual(normalize_author_list("M. d'Alembert & K.V.U. vanBeuren"), "d'Alembert, M.; vanBeuren, K.V.U.")
-        self.assertEqual(normalize_author_list("L. von Beethoven-Tschaikowski et al."), 'von Beethoven-Tschaikowski, L.')
-        self.assertEqual(normalize_author_list("Abazajian, K. N., Adelman-McCarthy, J. K."), 'Abazajian, K. N.; Adelman-McCarthy, J. K.')
+        self.assertEqual(normalize_author_list("Foo, N.A., Bar, C. K. "), 'Foo, N; Bar, C')
```

Comment:
ohoo, this made me smile -- because it looks like what ADS Classic author parsing rules would have me do in solr; my sincere condolences -- I suffered a lot from the same

but maybe it is just trying to improve recall....